### PR TITLE
[docs] Fix indentations in ai/get-started-macvlan

### LINKED ISSUE #4116: failed to cherry pick PR 4106 from windsonsea, to branch release-v1.0
commits bc053234a0a94dcdf180440c9ad31f35bfa73f1a of windsonsea conflict when merging to branch release-v1.0, please manually cherry pick it by yourself. 
 PR <https://github.com/spidernet-io/spiderpool/pull/4106> ,  action <https://github.com/spidernet-io/spiderpool/actions/runs/11044837239>  

  ```
CONFLICT (modify/delete): docs/usage/install/ai/get-started-macvlan-zh_CN.md deleted in HEAD and modified in bc053234 (Merge pull request #4106 from windsonsea/aiform).  Version bc053234 (Merge pull request #4106 from windsonsea/aiform) of docs/usage/install/ai/get-started-macvlan-zh_CN.md left in tree.
CONFLICT (modify/delete): docs/usage/install/ai/get-started-macvlan.md deleted in HEAD and modified in bc053234 (Merge pull request #4106 from windsonsea/aiform).  Version bc053234 (Merge pull request #4106 from windsonsea/aiform) of docs/usage/install/ai/get-started-macvlan.md left in tree.
error: could not apply bc053234... Merge pull request #4106 from windsonsea/aiform
hint: After resolving the conflicts, mark them with
hint: "git add/rm <pathspec>", then run
hint: "git cherry-pick --continue".
hint: You can instead skip this commit with "git cherry-pick --skip".
hint: To abort and get back to the state before "git cherry-pick",
hint: run "git cherry-pick --abort".
hint: Disable this message with "git config advice.mergeConflict false"
```

### PR DESCRIPTION
**Pull request by ty-dc**
#### What type of PR is this?

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #4116 

**Special notes for your reviewer**:

cherry-pick #4106

### CODE REVIEW

File: docs/usage/install/ai/get-started-macvlan-zh_CN.md [hunk 1]
```diff
@@ -0,0 +1,412 @@
...
+    ```shell
+    $ helm repo add spiderpool https://spidernet-io.github.io/spiderpool
+    $ helm repo update spiderpool
+    $ kubectl create namespace spiderpool
+    $ helm install spiderpool spiderpool/spiderpool -n spiderpool --set rdma.rdmaSharedDevicePlugin.install=true
+    ```
+
+    > 如果您是中国用户，可以指定参数 `--set global.imageRegistryOverride=ghcr.m.daocloud.io` 来使用国内的镜像源。
+
+   完成后，安装的组件如下
```
Comment (windsonsea) [inline docs/usage/install/ai/get-started-macvlan-zh_CN.md:155]:
```suggestion
    完成后，安装的组件如下：
```

File: docs/usage/install/ai/get-started-macvlan-zh_CN.md [hunk 2]
```diff
@@ -0,0 +1,412 @@
...
+
+    ```shell
+    # see 8 RDMA devices assigned to the Pod
+    $ rdma link
+
+    # Start an RDMA service
+    $ ib_read_lat
+    ```
+
+   开启一个终端，进入另一个 Pod 访问服务：
```
Comment (windsonsea) [inline docs/usage/install/ai/get-started-macvlan-zh_CN.md:404]:
```suggestion
    开启一个终端，进入另一个 Pod 访问服务：
```

**Review by windsonsea — COMMENTED** [review]
pick two remained

**Review by windsonsea — APPROVED** [review]
Some remained can be update later. This is a cherrypick.

**Review by windsonsea — APPROVED** [review]

**Review by weizhoublue — APPROVED** [review]

**Pull request merged**